packages/lesswrong/lib/utils/schemaUtils.ts

```diff
@@ -133,7 +138,7 @@ export function arrayOfForeignKeysField<CollectionName extends keyof Collections
   
   return {
     type: Array,
-    defaultValue: [],
+    ...schemaDefaultValue([]),
```

Comment:
We should double-check whether this substitution is safe, since it adds `onUpdate` as well as `onCreate` (and a couple other fields).

Reply:
If it's not, we'll need to explicitly specify `nullable: false` as a separate property